joy.py

```diff
@@ -490,6 +489,61 @@ class Group(Shape):
     def __init__(self, shapes, **kwargs):
         super().__init__("g", children=shapes, **kwargs)
 
+class Text(Shape):
+    """Creates a text node.
+
+    Parameters:
+        position:
+            The position of the text node.
+            Defaults to (0, 0) when not specified from the lower
+            left corner.
+
+    Examples:
+
+    Draw a text node:
+
+        >>> t = text("Hello!")
+        >>> show(t)
+
+    Draw a text node centered at (50, 50).
+
+        >>> t = text("Hello!", center=Point(x=100, y=100), alignment_baseline="middle", text_anchor="middle")
+        >>> show(t)
+    """
+    def __init__(self, content, anchor=Point(0, 0), **kwargs):
```

Comment:
This API is inconsitent with other shapes. We typically use `x` and `y` attributes to specify position instead of a `Point`. 

For example: `circle(x=100, y=100)`

I think it would be better to use it as `text("Hello", x=100, y=100)`